vault/identity_store_oidc_provider_test.go

```diff
@@ -1164,6 +1165,14 @@ func TestOIDC_Path_OIDC_ProviderClient(t *testing.T) {
 	if diff := deep.Equal(expected, resp.Data); diff != nil {
 		t.Fatal(diff)
 	}
+	clientID := resp.Data["client_id"].(string)
+	if !strings.HasPrefix(clientID, "id_") && len(clientID) != 32 {
+		t.Fatalf("client_id format is incorrect: %#v", clientID)
```

Comment:
Nit: If constants are used above, instead of having hardcoded values these could be 
```golang
len("id_") + ClientIDLength`
```

```golang
len("secret_") + ClientSecretLength`
```